tools/install-powershell.ps1

```diff
@@ -141,27 +156,43 @@ try {
         Write-Verbose "About to download package from '$downloadURL'" -Verbose
 
         $packagePath = Join-Path -Path $tempDir -ChildPath $packageName
+        if ($PSVersionTable.PSEdition -eq "Desktop") {
+            # On Windows PowerShell, progress can make the download significantly slower
+            $oldProgressPreference = $ProgressPreference
```

Comment:
`Invoke-WebRequest` can throw terminating error, so I would prefer resetting the `ProgressPreference` in a try-finally block.